src/Symfony/Bridge/ProxyManager/composer.json

```diff
@@ -19,7 +19,7 @@
         "php": ">=8.1",
         "friendsofphp/proxy-manager-lts": "^1.0.2",
         "symfony/dependency-injection": "^6.3",
-        "symfony/deprecation-contracts": "^2.1|^3"
+        "symfony/deprecation-contracts": "^2.5|^3"
```

Comment:
why bump deprecation-contracts? better not if not needed don't you think?

Reply:
For consistency, mainly. 2.5 is the release we still maintain in our LTS branch. Do you want to define a different target 2.x minor version for each contracts package? If so, let's define them and I adjust the PR.

Reply:
OK.
I looked for bumping to v3 but that'd make our packages less compatible with third party packages.